app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      log_in(@user)
+      flash[:success] = 'User Created'
+      redirect_to user_path(@user)
+    else
+      flash.now[:danger] = 'Some Errors in Form'
+      render 'new'
+    end
+  end
+
+  def show
+    @user = params[:id] ? User.find_by(id: params[:id]) : nil
+    return if @user
+
+    flash[:danger] = 'User not found'
+    redirect_to home_path
+  end
```

Comment:
- [ ] The events show page should be able to display a list of [attendees](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_show_requirement.png) and should [display](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) the events the user attended in the past and those that are upcoming.